src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "white";
+    private static final int RADIUS_DEFAULT = 10;
+    private static final int MAX_RANDOM = 20;
+    private Random random = new Random();
+
+    private int randomNumberForSide() {
+        return random.nextInt(MAX_RANDOM) + 1;
+    }
+
+    private String color() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
with such decision you will create new instance of ColorSupplier each time you call color() method
please move to class variable